recipes/pykafka/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pykafka" %}
+{% set version = "2.6.0" %}
+{% set sha256 = "19e4a802b00e28c3203fcfd80ca440b13faecb8e58171e9685a044cf87dae93e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - six >=1.5
+    - kazoo
+    - tabulate
+    - gevent
+    - librdkafka  # [not win]
+
+test:
+  imports:
+    - pykafka
+
+about:
+  home: http://github.com/parsely/pykafka
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Apache Kafka client for Python; high-level & low-level consumer/producer, with great performance'
+
+  description: |
+      PyKafka is a cluster-aware Kafka>=0.8.2 client for Python. It includes
+      Python implementations of Kafka producers and consumers, which are
+      optionally backed by a C extension built on librdkafka, and runs under
+      Python 2.7+, Python 3.4+, and PyPy.
+
+      PyKafka’s primary goal is to provide a similar level of abstraction to
+      the JVM Kafka client using idioms familiar to Python programmers and
+      exposing the most Pythonic API possible.
+  doc_url: http://pykafka.readthedocs.org/
+  dev_url: https://github.com/parsely/pykafka
+
+extra:
+  recipe-maintainers:
+    - mrocklin
+    - emmett9001
```

Comment:
Are you ok being a maintainer, @emmett9001?

Reply:
Yes I am!